corn/styles.css

```diff
@@ -0,0 +1,58 @@
+:root{
+    --font:'Inter', sans-serif;
+    --hr-color:#D0D0D0;
+    --font-size-p: 14px;
+    --font-size-h1:32px;
+    --font-weight-h1-h2:700;
+    --font-weight-p:600;
+    --font-size-html:18px;
+    --font-weight-li:500;
+    --background-color:#F8F8F8;
+    --font-color-h1:#0E0E11;
+    --color-subtitle-li:#575767;
+    --color-p:#B9B9BE;
+}
+
+html{
+    font-family: var(--font);
+    background-color: var(--background-color);
+    font-size: var(--font-size-html);
+}
+
+h1{
+    font-size: var(--font-size-h1);   
+    color: var(--font-color-h1);
+}
+
+h1, h2{
+    font-weight: var(--font-weight-h1-h2);
+}
+
+p{
```

Comment:
As a suggestion: I would create a special class cause rules for `<p>` is very generic.